meerk40t/lihuiyu/device.py

```diff
@@ -290,7 +290,7 @@ def move_speed(channel, _, speed, dx, dy, **kwgs):
             def move_at_speed():
                 yield "set", "speed", speed
                 yield "program_mode"
-                yield "move_relative", dx.mil, dy.mil
+                yield "move_rel", dx.length_mil, dy.length_mil
```

Comment:
Move at speed used move_relative which is private and should give physical units for move_rel